recipes/citeproc-py/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "citeproc-py" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/citeproc-py-{{ version }}.tar.gz
+  sha256: d9e3a224f936fe2e5033b5d9ffdacab769cedb61d96c4e0cf2f0b488f1d24b4e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - rnc2rng >=2.6.1,!=2.6.2
+  run:
+    - lxml
+    - python
+
+test:
+  source_files:
+    - tests
+  imports:
+    - citeproc
+  commands:
+    - pip check
+    - nosetests --where=tests --ignore-files=citeproc-test.py
+    - python tests/citeproc-test.py --no-update
+  requires:
+    - git
+    - nose
+    - pip
+
+about:
+  home: https://github.com/brechtm/citeproc-py
+  summary: Citations and bibliography formatter
+  license: 
+    - BSD-2-Clause
+    - CC-BY-SA-3.0
+    - MIT
```

This need to be included using SPDX expressions: https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions

```suggestion
  license: BSD-2-Clause AND CC-BY-SA-3.0 AND MIT
```

---

Ah, thanks!